src/components/Profile/components/MembershipsList/MembershipsList.module.scss

```diff
@@ -33,7 +33,7 @@
       padding-left: 0rem;
       justify-content: center;
       background-color: var(--mui-palette-primary-main);
-      color: var(--mui-palette-neutral-light);
+      color: var(--mui-palette-primary-contrastText);
       border-top-left-radius: 4px;
       border-top-right-radius: 4px;
```

Comment:
Since the header already has color, background-color, and border-radius set outside of this media query, the better fix would be to not overwrite those properties inside the media query. The media query should only change the properties that are specific to the smaller screen size.